data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/handler/ArtifactRequestHandler.java

```diff
@@ -161,7 +161,7 @@ public boolean canHandle(@NotNull MultipartRequest multipartRequest) {
                 .id(message.getId().toString())
                 .protocol(MessageProtocol.IDS_MULTIPART)
                 .dataDestination(dataDestination)
-                .connectorId(connectorId)
+                .connectorId(connectorId.toUri().toString())
```

Comment:
Maybe we can add a `toString` method to the `IdsId` to capsulated this call?

Reply:
done.